variables.tf

```diff
@@ -1,45 +1,10 @@
-
-variable "security_group_enabled" {
-  type        = bool
-  description = "Whether to create default Security Group for EFS."
-  default     = true
-}
-
-variable "security_group_description" {
-  type        = string
-  default     = "EFS Security Group"
-  description = "The Security Group description."
-}
-
-variable "security_group_use_name_prefix" {
-  type        = bool
-  default     = false
-  description = "Whether to create a default Security Group with unique name beginning with the normalized prefix."
-}
-
-variable "security_group_rules" {
-  type = list(any)
-  default = [
-    {
-      type        = "egress"
-      from_port   = 0
-      to_port     = 65535
-      protocol    = "-1"
-      cidr_blocks = ["0.0.0.0/0"]
-      description = "Allow all outbound traffic"
-    }
-  ]
-  description = <<-EOT
-    A list of maps of Security Group rules. 
-    The values of map is fully complated with `aws_security_group_rule` resource. 
-    To get more info see https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/security_group_rule .
-  EOT
-}
-
-variable "security_groups" {
+# Intentionally not deprecated via security_group_inputs.tf since it cannot effectively be replaced via var.additional_security_group_rules.
+# This is because the logic to create these rules exists within this module, and should not be passed in by the consumer
+# of this module.
+variable "allowed_cidr_blocks" {
```

Comment:
I will relent and allow us to keep `allowed_cidr_blocks` because it is easy enough to implement and it is convenient for users.